blobstore/list.go

```diff
@@ -76,7 +76,8 @@ func (bh *BlobHandler) HandleListByPrefix(c echo.Context) error {
 		}
 
 	}
-	if delimiter && !strings.HasSuffix(prefix, "/") {
+
+	if delimiter && prefix != "" && !strings.HasSuffix(prefix, "/") {
```

Comment:
Shouldnt this be after CheckAndAdjustPrefix like in the HandleListByPrefixWithDetails? It doesnt make sense that they are in two different places within the function